drivers/serial/uart_cc32xx.c

```diff
@@ -70,15 +72,21 @@ static int uart_cc32xx_init(struct device *dev)
 	MAP_UARTFIFODisable((unsigned long)config->base);
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
-	/* Clear any pending UART interrupts: we only care about RX, TX: */
-	MAP_UARTIntClear((unsigned long)config->base,
-		(UART_INT_RX | UART_INT_TX));
+	/* Clear any pending UART RX interrupts: */
+	MAP_UARTIntClear((unsigned long)config->base, UART_INT_RX);
 
 	IRQ_CONNECT(DT_TI_CC32XX_UART_4000C000_IRQ_0,
 		    DT_TI_CC32XX_UART_4000C000_IRQ_0_PRIORITY,
 		    uart_cc32xx_isr, DEVICE_GET(uart_cc32xx_0),
 		    0);
 	irq_enable(DT_TI_CC32XX_UART_4000C000_IRQ_0);
+
+	/* Fill the tx fifo, so Zephyr console & shell subsystems get "primed"
+	 * with first tx fifo empty interrupt when they first call
+	 * uart_irq_tx_enable().
+	 * Use the BEL character, which typically prints nothing to the console.
```

Comment:
Sorry, but BEL "typically" produces a sound. If someone really has such classical setup, and Zephyr rings on each boot to them, they will soon hate Zephyr ;-).

Reply:
It all depends on the terminal emulator - most don't ring a bell anymore.  I haven't found one that does. 
But I can change it to '\r'.